src/main/java/com/redhat/coolstore/model/ShoppingCart.java

```diff
@@ -21,7 +22,7 @@ public class ShoppingCart implements Serializable {
 	
 	private double cartTotal;
 			
-	private List<ShoppingCartItem> shoppingCartItemList = new ArrayList<ShoppingCartItem>();
+	private List<ShoppingCartItem> shoppingCartItemList = new ArrayList<>();
```

Comment:
Looks like it has some additional "linting" capabilities, like this, where it's eliminating that bit which is unnecessary (modern java versions don't need the type specification in the constructor if it's implied in the variable declaration)